src/bootstrap/lib.rs

```diff
@@ -396,13 +396,18 @@ impl Build {
         let src = config.src.clone();
         let out = config.out.clone();
 
+        #[cfg(unix)]
```

Comment:
I wasn't sure if I had to wrap the statement below in a block. Should I?

Reply:
This should be fine.